### PR TITLE
C4 a 513 validation changes

### PR DESCRIPTION
**Pull request by asifamingov**
Updated regex using the latest sheet provided by the Cyber Security team

### CODE REVIEW

**Review by DanielTurner — DISMISSED** [review]

File: src/validations/date-marks/best-date.js [hunk 1]
```diff
@@ -5,10 +5,9 @@
     valid = "The date mark must 20 characters or less.";
     return;
   }
-  if (/[|&;$%"<>()\{\}+\\]/.test(input)) {
-    valid = 'Please do not include html or special characters e.g. not any of these |&;$%"<>(){}+\/';
+  if (!/^\d{1,2}\/\d{1,2}\/\d{4}$/.test(input)) {
```
Comment (nishgakkhar) [inline src/validations/date-marks/best-date.js:8]: There is request to remove this, we don't need this validation. Please refer to this ticket's- C4A-521 comments

Comment (DanielTurner) [reply src/validations/date-marks/best-date.js:8]: Remove the validation? This is to stop injection, I believe that this should allow a date format but not other sql / mongosql injections.

Comment (nishgakkhar) [reply src/validations/date-marks/best-date.js]: Not removing per say but to modify, as the date field can have string of characters which is not in format 11/11/21 but can also be 'Apr 2020'

Comment (DanielTurner) [reply src/validations/date-marks/best-date.js]: :+1:  I didn't understand, maybe not enough :coffee:

**Review by nishgakkhar — APPROVED** [review]

**Pull request merged**